enterprise/tailnet/pgcoord_internal_test.go

```diff
@@ -411,6 +407,11 @@ func TestPGCoordinatorUnhealthy(t *testing.T) {
 	coordinator, err := newPGCoordInternal(ctx, logger, ps, mStore, mClock)
 	require.NoError(t, err)
 
+	mStore.EXPECT().UpdateTailnetPeerStatusByCoordinator(gomock.Any(), database.UpdateTailnetPeerStatusByCoordinatorParams{
+		CoordinatorID: coordinator.id,
+		Status:        database.TailnetStatusLost,
+	})
+
```

Comment:
this is an odd place for this `EXPECT()` statement.  The style of this test was to set up all expectations before starting to execute the test.  It should also have a comment explaining why it should happen and whether or not we care for the test.

Reply:
I figured it's always better to be more explicit (e.g. validate inputs) but given that the other tests cover that pretty well it's unnecessary.